config/migrations/2023/20230116120500-add-status-concepts.ttl

```diff
@@ -0,0 +1,24 @@
+
+
+<http://lblod.data.gift/concept-schemes/effb1058-d469-4064-a7a6-9d98ba081962> a <http://www.w3.org/2004/02/skos/core#ConceptScheme>;
+  <http://www.w3.org/2004/02/skos/core#prefLabel> "Statuses of public services";
+  <http://mu.semte.ch/vocabularies/core/uuid> "effb1058-d469-4064-a7a6-9d98ba081962".
+
+
+<http://lblod.data.gift/concepts/5a3168e2-f39b-4b5d-8638-29f935023c83> a <http://www.w3.org/2004/02/skos/core#Concept>;
+  <http://www.w3.org/2004/02/skos/core#inScheme> <http://lblod.data.gift/concept-schemes/effb1058-d469-4064-a7a6-9d98ba081962>;
+  <http://www.w3.org/2004/02/skos/core#prefLabel> "Updated";
+  <http://www.w3.org/2004/02/skos/core#topConceptOf> <http://lblod.data.gift/concept-schemes/effb1058-d469-4064-a7a6-9d98ba081962>;
+  <http://mu.semte.ch/vocabularies/core/uuid> "5a3168e2-f39b-4b5d-8638-29f935023c83".
+
+<http://lblod.data.gift/concepts/cf22e8d1-23c3-45da-89bc-00826eaf23c3> a <http://www.w3.org/2004/02/skos/core#Concept>;
+  <http://www.w3.org/2004/02/skos/core#inScheme> <http://lblod.data.gift/concept-schemes/effb1058-d469-4064-a7a6-9d98ba081962>;
+  <http://www.w3.org/2004/02/skos/core#prefLabel> "Deleted";
+  <http://www.w3.org/2004/02/skos/core#topConceptOf> <http://lblod.data.gift/concept-schemes/effb1058-d469-4064-a7a6-9d98ba081962>;
+  <http://mu.semte.ch/vocabularies/core/uuid> "cf22e8d1-23c3-45da-89bc-00826eaf23c3".
+
+<http://lblod.data.gift/concepts/176ec8a9-ffd6-4f0b-a302-84ef59b31c42> a <http://www.w3.org/2004/02/skos/core#Concept>;
+  <http://www.w3.org/2004/02/skos/core#inScheme> <http://lblod.data.gift/concept-schemes/effb1058-d469-4064-a7a6-9d98ba081962>;
+  <http://www.w3.org/2004/02/skos/core#prefLabel> "Reviewed";
```

Comment:
Thinking about it some more, do we actually need the "reviewed" status? Couldn't we simply interpret the absence of the triple as "no review required"?

That would also save us the work of having to set the "Reviewed" status on newly created public service instances.

Reply:
we could start with that yes